submissions/Temu4/js_dom/script.js

```diff
@@ -0,0 +1,75 @@
+  'Winchester rifle is a comprehensive term describing a series of lever-action repeating rifles manufactured by the Winchester Repeating Arms Company. Developed from the 1860 Henry rifle, Winchester rifles were among the earliest repeaters. The Model 1873 was particularly successful, being marketed by the manufacturer as "The Gun that Won the West".',
+  'A machine gun is a fully automatic mounted or portable firearm designed to fire rifle cartridges in rapid succession from an ammunition belt or magazine for the purpose of suppressive fire. Not all fully automatic firearms are machine guns. Submachine guns, rifles, assault rifles, battle rifles, shotguns, pistols or cannons may be capable of fully automatic fire, but are not designed for sustained fire. As a class of military rapid-fire guns, machine guns are fully automatic weapons designed to be used as support weapons and generally used when attached to a mount- or fired from the ground on a bipod or tripod. Many machine guns also use belt feeding and open bolt operation, features not normally found on rifles.',
+  'A sniper rifle is a high-precision rifle designed for sniper missions. It serves to fulfil the tactical need for long range surveillance, effective anti-personnel and anti-materiel operations with a high hit probability, and can be used by both military, law enforcement and civilian militias. The modern sniper rifle is a portable shoulder-fired weapon system with a choice between bolt-action or semi-automatic action, fitted with a telescopic sight for extreme accuracy and chambered for a high-ballistic performance centerfire cartridge.',
+  'A grenade is a large explosive weapon typically thrown by hand, but can also refer to projectiles shot out of grenade launchers. Generally, a grenade consists of an explosive charge, a detonating mechanism, and firing pin inside the grenade to trigger the detonating mechanism. Once the soldier throws the grenade, the safety lever releases, the striker throws the safety lever away from the grenade body as it rotates to detonate the primer. The primer explodes and ignites the fuze (sometimes called the delay element). The fuze burns down to the detonator, which explodes the main charge.',
+  'Ammunition (informally ammo) is the material fired, scattered, dropped or detonated from any weapon. Ammunition is both expendable weapons (e.g., bombs, missiles, grenades, land mines) and the component parts of other weapons that create the effect on a target (e.g., bullets and warheads). Nearly all mechanical weapons require some form of ammunition to operate.'
+];
+
+//Content elements
+const topicTitle = document.getElementById('topicTitle');
+const topicImg = document.getElementById('topicImg');
+const topicDescription = document.getElementById('topicDescription');
+
+const dataTypes = [
```

Comment:
Try to define you data in common object with pretty structure. Also you repeated types of weapons with `titles` array. DRY.

Reply:
fixed in [this](https://github.com/kottans/frontend-2019-p2p/pull/42/commits/58c5c1a4cfa68b1a1584c140de9a0010d8a48896) commit